packages/notification-services-controller/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export * as NotificationServicesController from './NotificationServicesController';
-export * as NotificationsServicesPushController from './NotificationServicesPushController';
+export * as NotificationServicesPushController from './NotificationServicesPushController';
```

Comment:
The notifications team is pursuing a different scheme with organizing their exports. It looks like they're planning to define export subpaths instead of enumerating named exports (see https://github.com/MetaMask/core/pull/4604).

Reply:
Yep, it's a bit messy for now. Once subpath exports get it, we'll clean up the imports used.